### DISCUSSION TITLE
API for Falcon Threat Surface Management

### DISCUSSION THREAD

**Discussion by seemplybogo** [CrowdStrike/falconpy#1034] [Q&A] [answered]
Hello :)

Is there an API, and\or support in falconpy for the "Falcon Threat Surface Management" module for EASM?

Thanks!!

**Comment by crowdstrikedcs** [comment, marked as answer]
Hi @seemplybogo thanks for the question!

Currently Falcon Surface has a separate API from the rest of the Falcon Platform. Documentation for this API can be found [here](https://app.reposify.com/api)

Since this API currently uses a unique API endpoint and authentication methodology from the rest of the Falcon Platform it has not yet been integrated with FalconPy. 

There are plans to move this API over in the future at which point it will be integrated into FalconPy.

Let us know with any more questions!
